pom.xml

```diff
@@ -165,6 +165,8 @@
         <configuration>
           <workingDirectory>frontend</workingDirectory>
           <installDirectory>target</installDirectory>
+          <!-- Skip this plugin step if skip property is set to true -->
+          <skip>${frontend.skip}</skip>
```

Comment:
Thanks so much for adding this! Fantastic idea, and I think it'll really pay off 💯